manifests/init.pp

```diff
@@ -311,6 +313,7 @@
   Stdlib::Port $dynflow_pool_size = $::foreman::params::dynflow_pool_size,
   String $jobs_service = $::foreman::params::jobs_service,
   Boolean $hsts_enabled = $::foreman::params::hsts_enabled,
+  Array[Stdlib::HTTPUrl] $cors_domains = $::foreman::params::cors_domains,
```

Comment:
HTTPUrl -> Httpurl

Reply:
Tests fail now with `unknown data type Stdlib::Httpurl`.

Reply:
I went with https://github.com/puppetlabs/puppetlabs-stdlib#stdlibhttpurl but that's not what's actually in the code :( https://github.com/puppetlabs/puppetlabs-stdlib/pull/1001 should fix that.